clients/roscpp/include/ros/spinner.h

```diff
@@ -109,10 +109,12 @@ class ROSCPP_DECL AsyncSpinner
 
 
   /**
-   * \brief Check if the spinner can be started. A spinner can't be started if
-   * another spinner is already running.
+   * \brief Check if the spinner can be started. The spinner shouldn't be started if
+   * another single-threaded spinner is already operating on the callback queue.
+   *
+   * This function is not necessary anymore. start() will always start spinning.
    */
-  bool canStart();
+  ROS_DEPRECATED bool canStart();
```

Comment:
Since we will not start building from that branch until early 2017  I don't see I reason to create that branch now. We can create a separate ticket so that the deprecation is not forgotten but done as soon as the branch exists.